package.json

```diff
@@ -84,7 +84,8 @@
     "react-window": "^1.8.5",
     "scplus-shared-components": "https://npm-public.selectquotelabs.com/scplus-shared-components/7.3.1",
     "text-mask-addons": "^3.8.0",
-    "use-debounce": "^6.0.0"
+    "use-debounce": "^6.0.0",
+    "yup": "^0.32.9"
```

Comment:
FWIW, there was a breaking change in Yup 0.31 that was "array().required() will no longer consider an empty array missing and required checks will pass." which we rely on for some of our components, name checkboxgroup. This is why your tests and ci are failing. So I think we need to ugprade yup in a seperate issue which will broke this and #395